includes/admin/class-amp-template-customizer.php

```diff
@@ -215,6 +237,24 @@ public function register_legacy_ui() {
 		do_action( 'amp_customizer_register_ui', $this->wp_customize );
 	}
 
+	/**
+	 * Get AMP panel description.
+	 *
+	 * This is also added to the root panel description in the AMP Customizer when a Reader theme is being customized.
+	 *
+	 * @return string Description, with markup.
+	 */
+	protected function get_amp_panel_description() {
+		return wp_kses_post(
+			sprintf(
+				/* translators: 1: URL to AMP project, 2: URL to admin settings screen */
+				__( '<a href="%1$s" target="_blank">AMP</a> is a project that emphasizes user experience. While AMP also applies to desktop pages, your site is <a href="%2$s" target="_blank">currently configured</a> in Reader mode to serve AMP pages to mobile visitors. These settings customize the experience for these users.', 'amp' ),
```

Comment:
Remove: `AMP is a project that emphasizes user experience`  (This message should conveyed during setup time; also, the use of `project` here is confusing)

Update: `While AMP works well on both desktop and mobile pages, your site is currently configured in Reader mode to serve AMP pages to mobile visitors.`